src/website/app/pages/Typography/index.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import colorable from 'colorable';
+import { createStyledComponent } from '../../../../utils';
+import Markdown from '../../Markdown';
+import {
+  Table as _Table,
+  TableCell,
+  TableHeaderCell,
+  TableRow
+} from '../../Table';
+import FontDemo from './FontDemo';
+import content from './typography.md';
+import examples from './examples';
+
+const a11yColor = color => {
```

Comment:
Not a blocker, but we should extract this into some kind of util. This is now the third (fourth?) place we use similar logic.